markdown/3.x/en/ff-asn.md

```diff
@@ -158,17 +160,19 @@ The `ff-slider-control` can have two `input` elements annotated with the `[data-
     </div>
 
     <ff-slider-control submit-on-input="true">
-        <ff-slider step-size="1" submit-on-release="true">
-            <div slot="slider1" class="sliderBtn"></div>
-            <div slot="slider2" class="sliderBtn"></div>
-        </ff-slider>
-
-        <div slot="sliderControls">
-            <input data-control="1" style="width: 60px;">
-            <input data-control="2" style="width: 60px;">
-        </div>
-
-        <div data-container="removeFilter">Reset Filter</div>
+       <div>
+           <ff-slider step-size="1" submit-on-release="true">
+               <div slot="slider1"></div>
+               <div slot="slider2"></div>
+           </ff-slider>
+           <div slot="sliderControls" style="display: flex; justify-content: space-around; align-items: center">
+               <input data-control="1" style="width: 60px;">
+               <span style="width: 20px; height: 2px; background-color: black; display: inline-block"></span>
+               <input data-control="2" style="width: 60px;">
+            </div>
+       </div>
```

Comment:
Was this additional `div` wrapper necessary to make it working? I remember there was some change in ff-slider-control related to this - just want to make sure why you changed the documentation here. And why added additional style to `sliderControls`? Did you just check if this example work on our controls? If so, it's OK then.

Reply:
yeah it worked before and works now as well.

Now the shown template is exactly the default template. I thought it is a nice easy extra information to write _"the templates default to the example below."_, as I added it.

Reply:
Ah, I see now. I missed the context.